crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -295,7 +318,9 @@ impl From<enums::CaptureMethod> for AuthorizationType {
     fn from(item: enums::CaptureMethod) -> Self {
         match item {
             enums::CaptureMethod::Manual => Self::Pre,
-            _ => Self::Final,
+            euclid::enums::CaptureMethod::Automatic
+            | enums::CaptureMethod::ManualMultiple
+            | enums::CaptureMethod::Scheduled => Self::Final,
```

Comment:
ManualMultiple and Scheduled is not supported by the connector. In such cases throw error